.azure-pipelines/generation-templates/java-kiota.yml

```diff
@@ -10,10 +10,6 @@ steps:
   env:
     MainDirectory: $(Build.SourcesDirectory)/${{ parameters.repoName }}/src/main/java/${{ parameters.namespacePath }}/generated/
 
-- pwsh : $(Build.SourcesDirectory)/${{ parameters.repoName }}/Scripts/incrementMinorVersion.ps1
```

Comment:
as we're using java v1 to test the new release automations, could we instead have a condition here that calls the script only for beta please?